arwen/elrondapi/elrondei.go

```diff
@@ -1380,15 +1381,12 @@ func v1_3_upgradeContract(
 		return
 	}
 
-	gasSchedule := metering.GasSchedule()
-	gasToUse = gasSchedule.ElrondAPICost.AsyncCallStep
```

Comment:
Are we sure we do not want to consume the async call gas?

Reply:
We are consuming it, but properly. See `runtime.ExecuteAsyncCall()` called below.